oasis-reference/src/main/resources/com.empowerops.volition.ref_oasis/OptimizerView.fxml

```diff
@@ -0,0 +1,173 @@
+                                    </columns>
+                                    <columnResizePolicy>
+                                        <TableView fx:constant="CONSTRAINED_RESIZE_POLICY" />
+                                    </columnResizePolicy>
+                                </TableView>
+                            </AnchorPane>
+                        </Tab>
+                        <Tab closable="false" text="Result Log">
+                            <AnchorPane>
+                                <TableView fx:id="resultTableView" prefHeight="200.0" prefWidth="750.0" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
+                                    <columns>
+                                        <TableColumn fx:id="resultSenderColumn" maxWidth="150.0" minWidth="150.0" prefWidth="150.0" text="Sender" />
+                                        <TableColumn fx:id="typeColumn" maxWidth="150.0" minWidth="150.0" prefWidth="150.0" text="Status" />
+                                        <TableColumn fx:id="inputColumn" text="Input" />
+                                        <TableColumn fx:id="outputColumn" text="Output" />
+                                    </columns>
+                                    <columnResizePolicy>
+                                        <TableView fx:constant="CONSTRAINED_RESIZE_POLICY" />
+                                    </columnResizePolicy>
+                                </TableView>
+                            </AnchorPane>
+                        </Tab>
+                    </TabPane>
+                </VBox>
+                <HBox>
+                    <Label fx:id="optimizerStatusLabel" text="Status" />
+                </HBox>
+            </VBox>
+        </bottom>
+        <center>
+            <VBox fx:id="selectedNodeInfoBox" spacing="5.0">
+                <VBox>
+                    <Label text="Configuration" />
+                    <HBox spacing="5.0">
+                        <Label text="Status: " />
+                        <Label fx:id="statusLabel" layoutX="10.0" layoutY="10.0" text="[Status]" />
+                    </HBox>
+                    <HBox spacing="5.0">
+                        <Label text="Description: " />
+                        <Label fx:id="descriptionLabel" layoutX="10.0" layoutY="10.0" text="[Description]" />
+                    </HBox>
+                    <HBox alignment="CENTER_LEFT" layoutX="10.0" layoutY="44.0" spacing="5.0">
+                        <CheckBox fx:id="useTimeout" />
+                        <Label text="Time Out:" />
+                        <TextField fx:id="timeOutTextField" />
+                        <Label text="in milliseconds" />
+                    </HBox>
+                    <Label layoutX="10.0" layoutY="10.0" text="Paramaters:" />
+                    <TreeTableView fx:id="paramTreeView">
```

Comment:
yeah this is a neat UI element. Its a tad complex, but it allows you to render both a table _and give users the efficiency of a tree index_ on that table. Very neat.